docs/paper/dev/api/scheduler.md

```diff
@@ -0,0 +1,207 @@
+﻿---
+slug: /dev/scheduler
+---
+
+# Using the Scheduler
+
+The `BukkitScheduler` can be used to schedule your code to be run later or repeatedly.
+
+:::info[Folia]
+
+This guide is designed for Non-Folia Bukkit Servers. If you are using Folia, you should use its respective schedulers.
+
+:::
+
+## What is a tick?
+
+Every game runs something called a game loop which essentially executes all the logic of the game over and over,
+a single execution of that loop is called a 'tick'.
+
+In Minecraft's case the amount of ticks per second is 20, or one tick every 50 milliseconds,
+meaning that the game loop is executed 20 times per second. A tick taking more than 50ms to execute is the moment
+when your server starts to fall behind on its work and lag.
+
+### Converting between human units and Minecraft ticks
+
+Every method of the scheduler that takes a delay or period uses ticks as a unit of time.
+
+Converting from human units to ticks and back is as simple as:  
+`ticks = seconds * 20`  
+`seconds = ticks / 20`
+
+You can make your code more readable by using the
+[`TimeUnit`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/concurrent/TimeUnit.html)
+enum, e.g. to convert 5 minutes to ticks and back:  
+`TimeUnit.MINUTES.toSeconds(5) * 20`  
+`TimeUnit.SECONDS.toMinutes(ticks / 20)`
+
+You can also use the `Tick` class from Paper to convert between human units and ticks, e.g. to convert 5 minutes to ticks:
+`Duration.ofMinutes(5).get(Tick.tick())` will yield `6000` ticks.
```

Comment:
This will throw a UnsupportedTemporalTypeException. Generally this method only support seconds and nanos "conversion". But you can uses:
from minutes to ticks:   Tick.tick().fromDuration(Duration.ofMinutes(5))
from ticks to minutes:   Tick.of(ticks).toMinutes()